Header header
string temp1
float32 temp2